ignite/engine/__init__.py

```diff
@@ -499,10 +500,6 @@ def supervised_evaluation_step_amp(
 
     .. versionadded:: 0.5.0
     """
-    try:
-        from torch.cuda.amp import autocast
-    except ImportError:
-        raise ImportError("Please install torch>=1.6.0 to use amp_mode='amp'.")
```

Comment:
To test this case without `from torch.cuda.amp import autocast`, please do as here: https://github.com/pytorch/ignite/blob/f29fc24699ee43c7acb4d212a2333565ec03f3a6/tests/ignite/engine/test_create_supervised.py#L485-L487